### PR TITLE
Fixed custom data type detection in --help and --help-all

### PR DESCRIPTION
**Pull request by pracucci**
#### What this PR does
While reviewing #2633, I've realized `model.Duration` and few other data types are not correctly detected when building `--help`. This PR fixes it.

#### Which issue(s) this PR fixes or relates to

N/A

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: pkg/util/usage/usage.go [hunk 1]
```diff
@@ -198,6 +198,20 @@ func getFlagName(fl *flag.Flag) string {
 		switch v.Type().String() {
 		case "*flagext.Secret":
 			return "string"
+		case "*flagext.StringSlice":
+			return "string"
+		case "*flagext.StringSliceCSV":
+			return "comma-separated list of string"
```
Comment (aknuds1) [inline pkg/util/usage/usage.go:204]:
```suggestion
			return "comma-separated list of strings"
```

File: pkg/util/usage/usage.go [hunk 2]
```diff
@@ -198,6 +198,20 @@ func getFlagName(fl *flag.Flag) string {
...
+		case "*flagext.CIDRSliceCSV":
+			return "comma-separated list of string"
```
Comment (aknuds1) [inline pkg/util/usage/usage.go:206]:
```suggestion
			return "comma-separated list of strings"
```

File: pkg/util/usage/usage.go [hunk 3]
```diff
@@ -198,6 +198,20 @@ func getFlagName(fl *flag.Flag) string {
...
+		case "*flagext.URLValue":
+			return "string"
+		case "*url.URL":
+			return "string"
+		case "*model.Duration":
+			return "duration"
+		case "*tsdb.DurationList":
+			return "comma-separated list of duration"
```
Comment (aknuds1) [inline pkg/util/usage/usage.go:214]:
```suggestion
			return "comma-separated list of durations"
```

**Review by aknuds1 — APPROVED** [review]
LGTM, except for some language nits.

**Comment by pracucci** [comment]
> LGTM, except for some language nits.

Sorry! I just noticed them after refreshing the GitHub page.

**Comment by pracucci** [comment]
> > LGTM, except for some language nits.
> 
> Sorry! I just noticed them after refreshing the GitHub page.

I'm working on a follow up PR to address these comments.

**Pull request merged**